assets/src/styles/blocks/Covers/layouts/TakeActionCovers.scss

```diff
@@ -95,186 +59,96 @@
+      z-index: 1;
     }
-  }
-
-  @include x-large-and-up {
-    padding: 32px 24px $n60;
-  }
-
-  &:before {
-    content: "";
-    opacity: 0;
-    transition: opacity 100ms linear;
-    position: absolute;
-  }
 
-  &:after {
-    content: "";
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-    background-image: linear-gradient(180deg, rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0));
-  }
-
-  &:hover, &:focus, &:focus-within {
-    @include background-before-opacity(linear-gradient(180deg, rgba(51, 51, 51, 0.6), rgba(250, 247, 236, 0.9), rgba(255, 255, 255, 0.8)), 1);
-    box-shadow: 0 0 10px transparentize($grey-80, .5);
-
-    @include medium-and-up {
-      p {
-        color: $grey-80;
-        display: block;
-      }
+    .cover-card-heading {
+      position: relative;
+      z-index: 1;
+      color: black;
+      font-size: 18px;
+      font-weight: 700;
+      margin: 8px 0;
+      line-height: 1.2;
+      display: -webkit-box;
+      -webkit-line-clamp: 2;
+      /* stylelint-disable property-no-vendor-prefix */
+      /*! autoprefixer: off */
+      -webkit-box-orient: vertical;
+      /*! autoprefixer: on */
+      /* stylelint-enable property-no-vendor-prefix */
+      overflow: hidden;
     }
 
-    .cover-card-btn {
-      display: block;
+    .cover-card-excerpt {
+      font-family: $lora;
+      font-size: 14px;
+      line-height: 1.5;
+      display: -webkit-box;
+      -webkit-line-clamp: 3;
+      margin: 0;
+      /* stylelint-disable property-no-vendor-prefix */
+      /*! autoprefixer: off */
+      -webkit-box-orient: vertical;
+      /*! autoprefixer: on */
+      /* stylelint-enable property-no-vendor-prefix */
+      overflow: hidden;
     }
   }
 
-  &--evergreen {
-    @include background-before-opacity($grey-80, .25);
+  .cover-card-btn {
+    font-size: 14px;
+    line-height: 1;
+    padding: 10px 20px;
+    right: 24px;
+    bottom: 24px;
+    left: auto;
+    position: absolute;
+    margin: 0;
+    border: none;
 
-    &:hover {
-      h2,
-      p {
-        color: $white;
-      }
+    html[dir="rtl"] & {
+      left: 24px;
+      right: auto;
     }
   }
 
-  &.action-card {
-    margin-bottom: $space-lg;
-  }
-
-  &.dark-card-bg {
-    color: $white;
-  }
-
-  & > * {
-    position: relative;
-    z-index: 1;
-  }
-
-  .cover-card-heading {
-    font-size: 1.5rem;
-    line-height: 1.2;
-    font-weight: 500;
-    margin-bottom: 24px;
-    padding-top: 0;
-    max-width: 80%;
-    transition: color 100ms linear;
-
-    --block-covers--heading-- {
-      color: $white;
-    }
-
-    text-shadow: 1px 1px 3px $grey-80;
-    display: table;
-    z-index: 1;
-    pointer-events: all;
-    font-family: $roboto;
+  @include medium-and-up {
+    min-height: 490px;
+    width: auto;
 
-    @include large-and-up {
-      font-size: 1.5rem;
-      margin-top: 8px;
-      max-width: 100%;
+    img {
+      height: 220px;
+      max-height: 220px;
+      min-height: 220px;
```

Comment:
Why do you need to set these attributes together?

Reply:
Hmmm yeah it's not ideal, it's because otherwise the `height` attribute gets overridden by the value set in [_page.scss](https://github.com/greenpeace/planet4-master-theme/blob/master/assets/src/scss/pages/_page.scss#L34). This could also be solved by using `!important` but I wanted to avoid that :/